test/armscan_env/test_labelmap_volumes.py

```diff
@@ -5,7 +5,8 @@
 import SimpleITK as sitk
 from armscan_env.clustering import TissueLabel
 from armscan_env.config import get_config
-from armscan_env.slicing import slice_volume
+from armscan_env.envs.state_action import ManipulatorAction
+from armscan_env.volumes.slicing import slice_volume
 
```

Comment:
I feel like some tests on the correctness of the transformations can be added. After addresssing the comment on separating sampling and transformation, you can actually write deterministic tests

Reply:
there is already an issue for that, and it is in the todos